config/accounts.go

```diff
@@ -1,9 +1,61 @@
 package config
 
+// RequestType : Request type enumeration
+type RequestType string
+
+// The request types
+const (
+	RequestTypeAMP   RequestType = "AMP"
+	RequestTypeApp   RequestType = "app"
+	RequestTypeVideo RequestType = "video"
+	RequestTypeWeb   RequestType = "web"
+)
+
 // Account represents a publisher account configuration
 type Account struct {
-	ID            string      `mapstructure:"id" json:"id"`
-	Disabled      bool        `mapstructure:"disabled" json:"disabled"`
-	CacheTTL      DefaultTTLs `mapstructure:"cache_ttl" json:"cache_ttl"`
+	ID            string      `mapstructure:"id"             json:"id"`
+	Disabled      bool        `mapstructure:"disabled"       json:"disabled"`
+	CacheTTL      DefaultTTLs `mapstructure:"cache_ttl"      json:"cache_ttl"`
 	EventsEnabled bool        `mapstructure:"events_enabled" json:"events_enabled"`
+	GDPR          AccountGDPR `mapstructure:"gdpr"           json:"gdpr"`
+}
+
+// AccountGDPR represents account-specific GDPR configuration
+type AccountGDPR struct {
+	Enabled            *bool                  `mapstructure:"enabled"             json:"enabled,omitempty"`
+	IntegrationEnabled AccountGDPRIntegration `mapstructure:"integration_enabled" json:"integration_enabled"`
+}
```

Comment:
Did GoFmt organize the attributes this way?

Reply:
"AMP" is "AMP" because it is an acronym, "Accelerated Mobile Pages" or something like that. The others are actual words.